deployer/test/deployment.test.js

```diff
@@ -146,8 +146,8 @@ describe('Main test', async () => {
     });
   });
 
-  // eslint-disable-next-line func-names
-  describe(`Adding ${numberOfBatches * batchSize} leaves in batches of ${batchSize}`, async function() {
+  describe(`Adding ${numberOfBatches * batchSize}
+    leaves in batches of ${batchSize}`, async () => {
```

Comment:
I've had tests fail on me for not writing `function() {` (rather than `() => {` in a `describe` block in the past. Do these tests run?

Reply:
changed code